src/components/primitive/box/box.component.tsx

```diff
@@ -2,8 +2,10 @@ import React from 'react';
 import { Col, Row } from 'react-grid-system';
 import styled, { useTheme } from 'styled-components';
 import _ from 'lodash';
+import { Oval } from 'react-loader-spinner';
 import { BoxComponentProps } from './box.component.props';
 import { spacing } from '../../../utils';
+import 'react-loader-spinner/dist/loader/css/react-spinner-loader.css';
```

Comment:
@cksadhukhan  Is there a better way to import this CSS? Getting compile error due to webpack I believe.